tox.ini

```diff
@@ -1,21 +1,26 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-{lint,unit}
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
 deps =
+    -rrequirements.txt
```

Comment:
This is fine, but it occurred to me that we include `requirements.txt` in `test-requirements.txt`, so that the experience involves a single `pip install`:

```
(tmp-bc3cc5ce42c72db) ✔ ~/.virtualenvs/tmp-bc3cc5ce42c72db
13:34 $ cat requirements.txt 
PyYAML
(tmp-bc3cc5ce42c72db) ✔ ~/.virtualenvs/tmp-bc3cc5ce42c72db 
13:35 $ cat test-requirements.txt 
-r requirements.txt
pytest
(tmp-bc3cc5ce42c72db) ✔ ~/.virtualenvs/tmp-bc3cc5ce42c72db 
13:35 $ pip install -r test-requirements.txt
Collecting PyYAML (from -r requirements.txt (line 1))
Collecting pytest (from -r test-requirements.txt (line 2))
  Using cached pytest-3.0.6-py2.py3-none-any.whl
Requirement already satisfied: setuptools in ./lib/python3.5/site-packages (from pytest->-r test-requirements.txt (line 2))
Collecting py>=1.4.29 (from pytest->-r test-requirements.txt (line 2))
  Using cached py-1.4.32-py2.py3-none-any.whl
Requirement already satisfied: appdirs>=1.4.0 in ./lib/python3.5/site-packages (from setuptools->pytest->-r test-requirements.txt (line 2))
Requirement already satisfied: six>=1.6.0 in ./lib/python3.5/site-packages (from setuptools->pytest->-r test-requirements.txt (line 2))
Requirement already satisfied: packaging>=16.8 in ./lib/python3.5/site-packages (from setuptools->pytest->-r test-requirements.txt (line 2))
Requirement already satisfied: pyparsing in ./lib/python3.5/site-packages (from packaging>=16.8->setuptools->pytest->-r test-requirements.txt (line 2))
Installing collected packages: PyYAML, py, pytest
Successfully installed PyYAML-3.12 py-1.4.32 pytest-3.0.6
```